doc/examples/html-handlebars.md

```diff
@@ -6,159 +6,159 @@ The example uses handlebars templates but can be easily adapted to other formats
+    var list = relatedNodes.map(function (node) {
+      return {
+        targetArrayString: JSON.stringify(node.target),
+        targetString: node.target.join(' ')
+      };
+    });
+    retVal += compiledRelatedListTemplate({relatedNodeList: list});
+  }
+  return retVal;
 }
- 
+
 function messagesFromArray(nodes) {
-	var list = nodes.map(function (failure) {
-		return {
-			message: failure.message.replace(/</gi, '&lt;').replace(/>/gi, '&gt;'),
-			relatedNodesMessage: messageFromRelatedNodes(failure.relatedNodes)
-		}
-	});
-	return compiledReasonsTemplate({reasonsList: list});
+  var list = nodes.map(function (failure) {
+    return {
+      message: failure.message.replace(/</gi, '&lt;').replace(/>/gi, '&gt;'),
+      relatedNodesMessage: messageFromRelatedNodes(failure.relatedNodes)
+    }
+  });
+  return compiledReasonsTemplate({reasonsList: list});
 }
- 
+
 function summary(node) {
-	var retVal = '';
-	if (node.any.length) {
-		retVal += '<h3 class="error-title">Fix any of the following</h3>';
-		retVal += messagesFromArray(node.any);
-	}
- 
-	var all = node.all.concat(node.none);
-	if (all.length) {
-		retVal += '<h3 class="error-title">Fix all of the following</h3>';
-		retVal += messagesFromArray(all);
-	}
-	return retVal;
+  var retVal = '';
+  if (node.any.length) {
+    retVal += '<h3 class="error-title">Fix any of the following</h3>';
+    retVal += messagesFromArray(node.any);
+  }
+
+  var all = node.all.concat(node.none);
+  if (all.length) {
+    retVal += '<h3 class="error-title">Fix all of the following</h3>';
+    retVal += messagesFromArray(all);
+  }
+  return retVal;
 }
- 
+
 /*
  * This code will generate a table of the rules that failed including counts and links to the Deque University help
  * for each rule.
  *
  * When used, you should attach click handlers to the anchors in order to generate the details for each of the
  * violations for each rule.
  */
- 
+
 if (results.violations.length) {
-	var violations = results.violations.map(function (rule, i) {
-		return {
-			impact: rule.impact,
-			help: rule.help.replace(/</gi, '&lt;').replace(/>/gi, '&gt;'),
-			bestpractice: (rule.tags.indexOf('best-practice') !== -1),
-			helpUrl: rule.helpUrl,
-			count: rule.nodes.length,
-			index: i
-		};
-	});
- 
-	html = compiledTableTemplate({violationList: violations});
+  var violations = results.violations.map(function (rule, i) {
+    return {
+      impact: rule.impact,
+      help: rule.help.replace(/</gi, '&lt;').replace(/>/gi, '&gt;'),
+      bestpractice: (rule.tags.indexOf('best-practice') !== -1),
+      helpUrl: rule.helpUrl,
+      count: rule.nodes.length,
+      index: i
+    };
+  });
+
+  html = compiledTableTemplate({violationList: violations});
 }
- 
+
 /*
  * To generate the human readable summary, call the `summary` function with the node. This will return HTML for that node.
  */
- 
+
 reasonHtml = summary(node);
 ```
 
 ## The Handlebars Templates
 
-```
+```javascript
```

Comment:
This should be HTML, no?

Reply:
Used the `handlebars` value since it appears valid for GFM.
I did find that VS Code didn't figure it out for the highlighting, but I'll see if I can find a repo to file that bug :)